desktop/package.js

```diff
@@ -13,6 +13,7 @@ const fs = require('fs-extra')
 const appName = 'Keybase'
 const shouldUseAsar = argv.asar || argv.a || false
 const shouldBuildAll = argv.all || false
+const shouldBuildArch = argv.arch || false
```

Comment:
This means "arch" as in "architecture" not as in "Arch Linux", right? Maybe worth being explicit just in case? (Very unfortunate choice of names on Arch Linux's part...)